epic_connection.py

```
@@ -0,0 +1,218 @@
+    local_server.bind(("127.0.0.1", LOCAL_PORT))
+    local_server.listen(1)
+    logger.info("Started local server")
+
+    size_data = 0
+
+    read_list = [local_server]
+
+    while True:
+        # allows for the while loop to not get stuck on the acceptance of
+        # socket connection and allowing the scheduling to get triggered
+        readable, writable, errored = select.select(read_list, [], [], 0)
```

Ruff (0.14.7)

Unpacked variable `writable` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)